src/Phinx/Db/Table.php

```diff
@@ -702,15 +702,9 @@ protected function executeActions($exists)
         // Renaming a table is tricky, specially when running a reversible migration
         // down. We will just assume the table already exists if the user commands a
         // table rename.
-        $renamed = collection($this->actions->getActions())
-            ->filter(function ($action) {
-                return $action instanceof RenameTable;
-            })
-            ->first();
-
-        if ($renamed) {
-            $exists = true;
-        }
+        $exists = $exists || count(array_filter($this->actions->getActions(), function ($action) {
```

Comment:
Revert this.

Reply:
semi-reverted so that it retains the same set-up, just using `current(array_filter(...))` instead of collection.